english/fill-sock/fill-sock.md

```diff
@@ -2,4 +2,6 @@ You quickly realise that marshmallows melt faster inside pink socks.
 
 This makes you decide to fill the sock with [more marshmallows](fill-sock.md).
 
-You decide [there's enough marshmallows for now](../marshmallow.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.